native/src/format.rs

```diff
@@ -1547,12 +1553,8 @@ pub fn format_defs(ps: &mut ParserState, defs: Defs) {
         }
 
         ps.emit_dot();
-        match ident_or_kw {
-            IdentOrKw::Ident(ident) => format_ident(ps, ident),
-            IdentOrKw::Kw(kw) => {
-                handle_string_and_linecol(ps, kw.1, kw.2);
-            }
-        }
+        let i = ident_or_kw.to_def_parts();
+        handle_string_and_linecol(ps, i.0, i.1);
```

Comment:
What do you think about adding some names here?

```suggestion
        let (ident, linecol) = ident_or_kw.to_def_parts();
        handle_string_and_linecol(ps, ident, linecol);
```